apps/project/views.py

```diff
@@ -621,7 +622,14 @@ def get(self, request, version=None):
         if fields_query:
             fields = fields_query.split(',')
 
-        options = {}
+        options = {
+            'organizations': [
```

Comment:
Shouldn't this be like this? Just to avoid confusion. Thanks.
```suggestion
            'project_organization_types': [
```